DataConnectors/esetSmc.json

```diff
@@ -0,0 +1,117 @@
+                                      },
+                                      "type": "InstallAgent"
+                                  }
+                              ]
+                          },
+                          {
+                              "title": "Install agent on a non-Azure Linux Machine",
+                              "description": "Download the agent on the relevant machine and follow the instructions.",
+                              "instructions": [
+                                  {
+                                      "parameters": {
+                                          "linkType": "InstallAgentOnLinuxNonAzure"
+                                      },
+                                      "type": "InstallAgent"
+                                  }
+                              ]
+                          }
+                      ]
+                  },
+                  "type": "InstructionStepsGroup"
+              }
+          ]
+      },
+      {
+          "title": "2. Configure the logs to be collected",
+          "description": "Configure rsyslog to accept logs from your Eset SMC IP address.\n\n```\nsudo -i\r\n\r\n# Set ESET SMC source IP address\r\nexport ESETIP=1.2.3.4\r\n\r\n# Create rsyslog configuration file\r\ncat > \/etc\/rsyslog.d\/80-remote.conf << EOF\r\n\\$ModLoad imudp\r\n\\$UDPServerRun 514\r\n\\$ModLoad imtcp\r\n\\$InputTCPServerRun 514\r\n\\$AllowedSender TCP, 127.0.0.1, $ESETIP\r\n\\$AllowedSender UDP, 127.0.0.1, $ESETIP\r\nuser.=alert;user.=crit;user.=debug;user.=emerg;user.=err;user.=info;user.=notice;user.=warning  @127.0.0.1:25224\r\nEOF\r\n\r\n# Restart rsyslog\r\nsystemctl restart rsyslog```"
+      },
+      {
+          "title": "3. Configure OMS agent to pass EsetSMC data in API format",
+          "description": "In order to easily recognize Eset data we will push it to separated table. To make it simple we will just modify match oms. section to send data as API objects by changing type to out_oms_api. Modify file on \/etc\/opt\/microsoft\/omsagent\/(REPLACEyourworkspaceid)\/conf\/omsagent.conf. Full match oms. section looks like this:\r\n\r\n```\r\n<match oms.** docker.**>\r\n  type out_oms_api\r\n  log_level info\r\n  num_threads 5\r\n  run_in_background false\r\n\r\n  omsadmin_conf_path \/etc\/opt\/microsoft\/omsagent\/2ce631ed-7e1a-4fe4-acdc-d8594412519c\/conf\/omsadmin.conf\r\n  cert_path \/etc\/opt\/microsoft\/omsagent\/2ce631ed-7e1a-4fe4-acdc-d8594412519c\/certs\/oms.crt\r\n  key_path \/etc\/opt\/microsoft\/omsagent\/2ce631ed-7e1a-4fe4-acdc-d8594412519c\/certs\/oms.key\r\n\r\n  buffer_chunk_limit 15m\r\n  buffer_type file\r\n  buffer_path \/var\/opt\/microsoft\/omsagent\/2ce631ed-7e1a-4fe4-acdc-d8594412519c\/state\/out_oms_common*.buffer\r\n\r\n  buffer_queue_limit 10\r\n  buffer_queue_full_action drop_oldest_chunk\r\n  flush_interval 20s\r\n  retry_limit 10\r\n  retry_wait 30s\r\n  max_retry_wait 9m\r\n<\/match>\r\n```\r\n"
```

Comment:
Why are we doing this? Why not push to Syslog table and just snap to the Syslog template format? Why converting Syslog to REST API? Also if you push into Syslog you can use the conversion to REST API as a parser/Kusto Function to well format and enable customers to work with the data.